src/components/SignUp/Email/EmailForm/EmailForm.tsx

```diff
@@ -0,0 +1,40 @@
+"use client"
+
+import * as styles from "./emailForm.css";
+import React, { useState } from "react";
+import TextField from "@/components/Common/TextField/TextField";
+import { SubmitButton } from "@/components/SignUp/Email/SubmitButton/SubmitButton";
+
+function EmailForm() {
+  const [email, setEmail] = useState("");
+  const [isTyping, setIsTyping] = useState(false);
+
+  const isValidEmail = (email: string) => /^[a-zA-Z0-9._%+-]+@kyonggi\.ac\.kr$/.test(email);
```

Comment:
`email` state의 유효성을 검증하면 되니까 함수로 만들지 않아도 될 것 같네요!!
```suggestion
  const isValidEmail = /^[a-zA-Z0-9._%+-]+@kyonggi\.ac\.kr$/.test(email);
```

Reply:
++ 정규 표현식에 익숙하지 않다면 스키마 기반으로 직관적이고 간편한 유효성 검사를 제공하는 `zod`라는 라이브러리를 도입해봐도 좋을 것 같아요~!

https://github.com/colinhacks/zod

Reply:
오호라... 틴유에서 유효성 검사는 이메일 입력하는 텍스트필드에만 필요한 것 같은데 isValidEmail 로 해결하기는 어려울까용?? 저는 도입 찬성파입니당 뭐든 해보쟈구요. 🌝

Reply:
오홍 그렇군여 그럼 지금은 당장 필요 없을 것 같기도 해요!!
그렇지만 추후 스프린트에 다른 학교들도 추가된다면 각 학교의 이메일 도메인이 계속 추가되고 검증 로직이 복잡해질텐데, 그런 측면에서 스키마 기반으로 검증을 하는 게 유지보수 측면에서 더 이점을 가질 수도 있습니다!!!

Reply:
오... 거기까지 생각못했핑이네요 귯 제안입니다 긍정검토👏✨